index.js

```diff
@@ -23,6 +23,16 @@ function minifier(safe_words, mutate_storages) {
   return through(mutate)
 
   function mutate(node) {
+    // remove unnecessary grouping operators
+    if(is_unnecessary_group(node)) return
+
+    if(node.parent) for(var current = node.parent; current.parent; current = current.parent) {
```

Comment:
For stylistic/readability purposes, can you give this `if` statement braces, and encapsulate the `for` loop within it?